docker-compose/technical/grafana/dashboards/alerts_summary.json

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
It would be nice to add an autorefresh to this dashboard. I'm still not sure why we need to reference a datasource "prometheus" here as it's only Grafana stuff. Can we try importing this dashboard on devopf?